aten/src/ATen/native/Copy.cpp

```diff
@@ -160,6 +160,16 @@ static Tensor & copy_impl(Tensor & self, const Tensor & src, bool non_blocking)
     return self;
   }
 
+  // Copies into meta self are OK and just ignored (similar to inplace)
+  if (self.is_meta()) {
+    // TODO: need to see if there is extra error checking needed
+    return self;
```

Comment:
I see some specialization stuff going on above this code for FBGEMM. Do we need to worry about meta tensors being used with FBGEMM enabled? It might be worth moving this check further up in case people do funky early-returns closer to the top of the function 😛

Reply:
it's safe because fbgemm only applies when it's CPU. But even better would be to actually get dispatch keys working correctly on this function